src/Microsoft.DotNet.Build.Tasks/PackageFiles/MSBuild.runtimeconfig.json

```diff
@@ -2,7 +2,7 @@
   "runtimeOptions": {
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "1.0.0-rc3-002733"
```

Comment:
Agreed, unfortunately this blocks the core-setup fix because it needs to update CLI as part of getting builds working again. (And there's no reason to merge this PR if it doesn't fix core-setup.) Good to know about those extra steps, thanks: my Linux build with this package works, but the Windows one doesn't, and that second point might be the cause. I haven't looked very far into the Windows error yet, though.